src/api-v3/paths/transactions/{transaction_id}.ts

```diff
@@ -29,16 +29,9 @@ export default function(api: RippleAPI, log: Function): Operations {
         res.status(status).json(response);
       });
     } else {
-      const params: any = {
+      return api.request('tx', Object.assign({}, {
         transaction: req.params.transaction_id
-      }
-      if (req.query.min_ledger) {
-        params.min_ledger = req.query.min_ledger;
-      }
-      if (req.query.max_ledger) {
-        params.max_ledger = req.query.max_ledger;
-      }
-      return api.request('tx', params).then((info: AppliedTransaction) => {
+      }, req.query)).then((info: AppliedTransaction) => {
```

Comment:
This diff is slightly hard to read, but instead of creating a params object on its own, now we just use `Object.assign` and include _all_ of the query parameters.